core/src/main/java/com/graphhopper/util/DouglasPeucker.java

```diff
@@ -53,6 +56,17 @@ public DouglasPeucker setMaxDistance(double dist) {
         return this;
     }
 
+    /**
+     * Weight to give elevation changes during simplification:
+     * 0 means elevation is ignored during simplification
+     * 0.2 means that 5 meters of elevation change is treated like
+     * 1m of lat/lon change
+     */
+    public DouglasPeucker setElevationFactor(double factor) {
```

Comment:
Do we want to stay with this unit and name for the elevation factor param? I had thought about making it either elevation max dist (with unit meters) or just inverse of what this is now.

Reply:
The absolute value (elevation max dist in meters) makes sense and would be preferred (I initially did not understand it ... see above ;) ). But I guess there is a reason why you chose it as a percentage?

Reply:
That makes sense, I switched everything over to "elevation max dist" - elevation factor was an attempt to pass through directly what the calculation needed, but max distance makes more sense for users.